_data/schedule.yml

```diff
@@ -1,5 +1,100 @@
 2016:
   dates: March 10-11 2016
+  programme:
+      - day: Wednesday evening, 9th March
+      schedule:
+        - time: 5:30 pm - 9:30 pm
+          event: Pre-conference drinks
+          description: >
+            Come join us for pre-conference drinks at
+            <a href="http://dropkickmurphys.co.za/">Dropkick Murphy's</a>, sponsored by
```

Comment:
What about https://www.facebook.com/dropkickmurphyscpt/ ?
Their website only has the durban address.